components/forgerock.key.manager/src/main/java/org/wso2/forgerock/client/ForgerockOAuthClient.java

```diff
@@ -0,0 +1,748 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
2021

Reply:
Fixed in 0cbcfc879bd87c85a4c14d959c2d28653f7fd391